src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionCoordinator.swift

```diff
@@ -105,7 +105,14 @@ final class ExposureDetectionCoordinator {
 	}
 	
 	private func showRiskOfContagionInfoScreen() {
-		// TODODODO: Navigate
+		let infoNavigationController = UINavigationController()
```

Comment:
Why not `init(rootViewController:)` would make the`infoNavigationController.pushViewController(viewController, animated: false)` in line 114 obsolete.

Reply:
done